common/src/vr.rs

```diff
@@ -32,7 +32,7 @@ pub struct VrUpdate {
 
```

Comment:
^ After looking at the definition of VrUpdate, perhaps change the name and add documentation to this. VR update message is quite vague.

Reply:
It's ALL bad. TODO: Fix this